dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -623,12 +623,12 @@
     </dependency>
     <dependency>
       <groupId>io.opentelemetry</groupId>
-      <artifactId>opentelemetry-exporter-otlp-common</artifactId>
+      <artifactId>opentelemetry-exporter-logging</artifactId>
```

Comment:
Is this a change you made? Looks like it might just be a merge inclusion...

Reply:
Removed